server/sonar-server/src/main/java/org/sonar/server/db/migrations/PlatformDatabaseMigration.java

```diff
@@ -97,18 +99,38 @@ public void run() {
         startDate = new Date();
         failureError = null;
         try {
-          LOGGER.info("Starting DB Migration at {}", startDate);
-          rubyBridge.databaseMigration().trigger();
-          LOGGER.info("DB Migration ended successfully at {}", new Date());
+          upgradeDb();
+          restartContainer();
+          recreateWebRoutes();
           status = Status.SUCCEEDED;
         } catch (Throwable t) {
-          LOGGER.error("DB Migration failed and ended at " + startDate + " with an exception", t);
+          LOGGER.error(
+            "DB Migration or container restart failed. Process ended at " + startDate + " with an exception", t
+            );
           status = Status.FAILED;
           failureError = t;
         } finally {
           running.getAndSet(false);
         }
       }
+
+      private void upgradeDb() {
+        LOGGER.info("Starting DB Migration at {}", startDate);
+        rubyBridge.databaseMigration().trigger();
+        LOGGER.info("DB Migration ended successfully at {}", new Date());
+      }
+
+      private void restartContainer() {
+        LOGGER.info("Restarting container at {}", new Date());
+        platform.doStart();
+        LOGGER.info("Container restarted successfully at {}", new Date());
+      }
+
+      private void recreateWebRoutes() {
+        LOGGER.info("Recreating web routes at {}", new Date());
+        rubyBridge.railsRoutes().recreate();
+        LOGGER.info("Routes recreated successfully at {}", new Date());
```

Comment:
Not sure such logs are needed, particularly the explicit end date as it's the first column of any log.